### PR TITLE
Tortuga-class Converted Cargo Freighter

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request

tortuga
imagen
imagen
imagen

Originally a cargo transport vessel, the Tortuga class is an improvised combat vessel under the ranks of the Frontiersmen fleet. Clever field modifications include the conversion of the cargo bay to a hangar, and additional padding around the ship in the form of asteroid rocks. It serves a dual purpose, both as a capable ship for establishing a military enclave in a sector, and as a sturdy vessel for reconnaissance. The nimble Brawler-class subshuttle of the Tortuga is offset by its very low overall mobility and loss of operation if the subshuttle were to be lost.

It is staffed by:
* 1 Captain
* 1 Shipswain (Head of Security)
* 1 Aidman
* 1 Sapper
* 2 Boarders
* 2 Rookies

## Why It's Good For The Game

The Ember is a pseudo-dreadnaught, and unfortunately, it kind of sucks. Pirates need a subshuttle to effectively raid.

## Changelog

:cl:
add: Tortuga-class.
add: Brawler-class.
del: Ember-class.
del: Scar suit.
tweak: Frontiersmen outfits start with less stuff.
/:cl:

### CODE REVIEW

File: _maps/configs/pirate_tortuga.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Tortuga-class Converted Cargo Freighter",
+	"prefix": "ISV",
+	"namelists": [
+		"PIRATES",
+		"BRITISH_NAVY"
+	],
+	"map_short_name": "Tortuga-class",
+	"map_path": "_maps/shuttles/pirate/pirate_tortuga.dmm",
+	"description": "Originally a cargo transport vessel, the Tortuga class is an improvised combat vessel under the ranks of the Frontiersmen fleet. Clever field modifications include the conversion of the cargo bay to a hangar, and additional padding around the ship in the form of asteroid rocks. It serves a dual purpose, both as a capable ship for establishing a military enclave in a sector, and as a sturdy vessel for reconnaissance. The nimble Brawler-class subshuttle of the Tortuga is offset by its very low overall mobility and loss of operation if the subshuttle were to be lost.",
```
Comment (Apogee-dev) [inline _maps/configs/pirate_tortuga.json]: "Originally a cargo transport vessel, the Tortuga is an improvised combat vessel used by the Frontiersmen fleet. Civilian transports are modified with shuttle handling facilities, and asteroid rubble is belted to the exterior of the hull as a form of cheap improvised armor and concealment. The Tortuga serves a dual purpose as a reconnaissance element and a vessel capable of establishing forward bases within a sector. While the Brawler-class shuttle carried by the Tortuga offers a surprising degree of speed and range to its boarding parties, the Tortuga itself is sluggish and nearly helpless without its shuttle."

Several edits for clarity and grammar.

File: _maps/configs/pirate_tortuga.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			"outfit": "/datum/outfit/job/frontiersmen/captain",
+			"officer": true,
+			"slots": 1
+		},
+		"Shipswain": {
+			"outfit": "/datum/outfit/job/frontiersmen/hos",
+			"officer": true,
+			"slots": 1
+		},
+		"Aidman": {
```
Comment (Apogee-dev) [inline _maps/configs/pirate_tortuga.json]: I feel like we could probably come up with a better name for this, "Aidman" is ridiculously clunky. Ship's Surgeon? Medic? It doesn't need to be completely unique to the faction.

File: _maps/configs/pirate_tortuga.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		"Subshuttle"
+	],
+	"limit": 1,
+	"job_slots": {
+		"Captain": {
+			"outfit": "/datum/outfit/job/frontiersmen/captain",
+			"officer": true,
+			"slots": 1
+		},
+		"Shipswain": {
```
Comment (Apogee-dev) [inline _maps/configs/pirate_tortuga.json]: Why not just use the more common "Bosun" or "Boatswain" here? Both roll off the tongue quite a bit better. You could even use Deck Boss or something if you want to keep it from being too age of sail-y.

File: _maps/configs/pirate_tortuga.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+			"outfit": "/datum/outfit/job/frontiersmen/hos",
+			"officer": true,
+			"slots": 1
+		},
+		"Aidman": {
+			"outfit": "/datum/outfit/job/frontiersmen/doctor",
+			"slots": 1
+		},
+		"Sapper": {
```
Comment (Apogee-dev) [inline _maps/configs/pirate_tortuga.json]: Another awkard job name, given the sapper's primary job is actually going to be maintaining the ship, with occasional assistance for breaches. I'd just go with Mechanic or something similar. As with the doctor slot above, you don't need to use completely faction-unique names for every role, as long as they fit the faction's flavor and suit the job. You also might want to give them a second engineer if you intend for one to join the boarders on the shuttle.

**Review by Apogee-dev — DISMISSED** [review]

**Review by Latentish — DISMISSED** [review]
Subshuttle has a few issues. It doesn't dock in the ship correctly, the doors have problems opening, and shield gens are fucky.

**Comment by Latentish** [comment]
This has that abandoned warehouse barren "you live like this" dollar store meth lab type feel.  Pretty good. Wanna tm this again to  make sure it's all working properly.

**Pull request merged**